lib/tomo/plugin/nodenv/tasks.rb

```diff
@@ -0,0 +1,63 @@
+require "shellwords"
+
+module Tomo::Plugin::Nodenv
+  class Tasks < Tomo::TaskLibrary
+    def install
+      run_installer
+      modify_bashrc
+      compile_node
```

Comment:
Maybe rename this to `install_node` since unlike ruby-build, in most cases node-build is not actually compiling from source.